nodejs/week1/home/avg.js

```diff
@@ -0,0 +1,19 @@
+let sum = 0;
+
+if (process.argv.length === 2) {
+    console.log("Please enter a number");
+    return 0;
+}
+
+for (let i = 2; i < process.argv.length; i++) {
+    for (let j = 2; j < process.argv.length; j++) {
+        if (isNaN(process.argv[j])) {
+            console.log("Please enter a number");
+            return 0;
+        }
+    }
```

Comment:
This `for` loop should only be run once, as your arguments do not change over time. Therefore it is enough to run this `for` loop once before the outer `for` loop, no need to execute it for each value of `i`.